mcs/class/System.Configuration/System.Configuration/SectionInformation.cs

```diff
@@ -201,12 +203,6 @@ public void UnprotectSection ()
 			protection_provider = null;
 		}
 
-		public void SetRawXml (string rawXml)
```

Comment:
This is public API. You cannot just remove it.

Reply:
My fault. Apologize.